### PR TITLE
Feature/lnguyen cone shape feature bdt

### PR DESCRIPTION
**Pull request by VCLanNguyen**
- add Cone feature to Track Shower ID MVA
- enable Cone feature in LArContent.cc

### CODE REVIEW

**Comment by VCLanNguyen** [comment]
Fixing comments from https://github.com/PandoraPFA/LArContent/pull/190

**Comment by VCLanNguyen** [comment]
Fixed comments from https://github.com/PandoraPFA/LArContent/pull/190

**Review by absolution1 — APPROVED** [review]
Looks good to me, thanks @VCLanNguyen

File: larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc [hunk 1]
```diff
@@ -298,6 +298,129 @@ StatusCode PfoHierarchyFeatureTool::ReadSettings(const TiXmlHandle /*xmlHandle*/
 //------------------------------------------------------------------------------------------------------------------------------------------
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+ConeChargeFeatureTool::ConeChargeFeatureTool(): m_conMinHits(3), m_minCharge (0.1f), m_conFracRange(0.2f), m_MoliereRadius(10.1f), m_MoliereRadiusFrac(0.2f)
```
Comment (AndyChappell) [inline larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc:301]: Where does the default value for the Moliere radius come from? I thought this was about 9cm for liquid Argon

File: larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.h [hunk 2]
```diff
@@ -130,7 +130,9 @@ class PfoHierarchyFeatureTool : public PfoCharacterisationFeatureTool
     void Run(LArMvaHelper::MvaFeatureVector &featureVector, const pandora::Algorithm *const pAlgorithm, const pandora::ParticleFlowObject *const pInputPfo);
 
 private:
+
```
Comment (AndyChappell) [inline larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.h:133]: Remove blank line

**Review by AndyChappell — CHANGES_REQUESTED** [review]
Conceptually it all looks good to me, a few minor comments, but otherwise this looks ready to go to me. Thanks!

File: larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc (hunk 1 above)
Comment (VCLanNguyen) [reply larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc:301]: I took it directly from the warwick pid: https://github.com/LArSoft/larana/blob/develop/larana/ParticleIdentification/MVAAlg.cxx#L546-L547
I cross-check on https://lar.bnl.gov/properties/#refs and I think it's about 10.0 for LAr @E=0.5kV

**Comment by VCLanNguyen** [comment]
> Conceptually it all looks good to me, a few minor comments, but otherwise this looks ready to go to me. Thanks!

Addressed comments from Andy

**Pull request merged**